guidelines/terms/20/synchronized-media.html

```diff
@@ -5,4 +5,5 @@
       interactive components, unless the media is a <a>media alternative for text</a> that is clearly labeled as such
    </p>
    
+   <p class="note">"Synchronized" refers to all audio and video content presented in the same multimedia experience (whether combined in a single file or played in the same time base). Images and sounds that are out of synch with each other (intentionally or unintentionally) are included by the definition. Asynchronous sound, for example, is still being experienced in the context of time-based, synchronized media.</p>
```

Comment:
```suggestion
   <p class="note">"Synchronized" refers to all audio and video content presented in the same multimedia experience (whether combined in a single file or played at the same time). Images and sounds that are out of synch with each other (intentionally or unintentionally) are included by the definition. Asynchronous sound, for example, is still being experienced in the context of time-based, synchronized media.</p>
```

Reply:
related https://github.com/w3c/wcag/issues/4398

Reply:
"this also hinges of whether or not there's actual important content/information being conveyed" How is importance relevant? Importance matters in what gets audio described, but seem irrelevant to a discussion on audio-video synchronization.

Reply:
Muzak is still information. There is _no_ normative language here about "important". It's just 'information'. I think folks are cross-circuiting the 'important visual' concept in audio descriptions with the notion of synchronized, which is just an amalgam of two time-based media.

Reply:
I'm going to make this an additional note to Audio Description, as discussed, leaving the Synchronization definition alone for now